src/routes/volume/CloneVolume.js

```diff
@@ -0,0 +1,274 @@
+                  },
+                },
+              ],
+            })(<InputNumber disabled style={{ width: '330px' }} />)}
+          </FormItem>
+          <FormItem style={{ flex: '1 0 30%' }}>
+          {getFieldDecorator('unit', {
+            initialValue: volume.unit || 'Gi',
+            rules: [{ required: true, message: 'Please select your unit!' }],
+          })(
+            <Select
+              disabled
+              style={{ width: '100px' }}
+            >
+              <Option value="Mi">Mi</Option>
+              <Option value="Gi">Gi</Option>
+            </Select>,
+          )}
+          </FormItem>
+        </div>
+        <FormItem label="Number of Replicas" hasFeedback {...formItemLayout}>
+          {getFieldDecorator('numberOfReplicas', {
+            initialValue: volume.numberOfReplicas,
+            rules: [
+              {
+                required: true,
+                message: 'Please input the number of replicas',
+              },
+              {
+                validator: (rule, value, callback) => {
+                  if (value === '' || typeof value !== 'number') {
+                    callback()
+                    return
+                  }
+                  if (value < 1 || value > 10) {
+                    callback('The value should be between 1 and 10')
+                  } else if (!/^\d+$/.test(value)) {
+                    callback('The value must be a positive integer')
+                  } else {
+                    callback()
+                  }
+                },
+              },
+            ],
+          })(<InputNumber />)}
+        </FormItem>
+        <FormItem label="Frontend" hasFeedback {...formItemLayout}>
+          {getFieldDecorator('frontend', {
+            initialValue: volume.frontend || frontends[0].value,
+            rules: [
+              {
+                required: true,
+                message: 'Please select a frontend',
+              },
+            ],
+          })(<Select>
+          { frontends.map(opt => <Option key={opt.value} value={opt.value}>{opt.label}</Option>) }
+          </Select>)}
+        </FormItem>
+        <FormItem label="Data Locality" hasFeedback {...formItemLayout}>
+          {getFieldDecorator('dataLocality', {
+            initialValue: volume.dataLocality || defaultDataLocalityValue,
+          })(<Select>
+          { defaultDataLocalityOption.map(value => <Option key={value} value={value}>{value}</Option>) }
+          </Select>)}
+        </FormItem>
+        <FormItem label="Access Mode" hasFeedback {...formItemLayout}>
+          {getFieldDecorator('accessMode', {
+            initialValue: volume.accessMode || 'rwo',
+          })(<Select>
+            <Option key={'ReadWriteOnce'} value={'rwo'}>ReadWriteOnce</Option>
+            <Option key={'ReadWriteMany'} value={'rwx'}>ReadWriteMany</Option>
+          </Select>)}
+        </FormItem>
+        <FormItem label="Backing Image" hasFeedback {...formItemLayout}>
```

Comment:
Hi @PhanLe1010 or @ChanYiLin , is there any meaning that user choose other backing image while cloning the volume ? 
Can we disable or remove this field in volume clone modal ?

<img width="698" alt="Screenshot 2024-06-27 at 12 06 24 PM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/0d4eb9ae-fae2-4dac-8c00-73d30bcb7570">

Reply:
After discussed with @ChanYiLin , we will disabled this backing image parameter while cloning volume.